WalletWasabi.Fluent/Views/Wallets/Send/SendFeeView.axaml

```diff
@@ -161,6 +161,10 @@
             <Setter Property="ToolTip.HorizontalOffset" Value="-30" />
           </Style>
         </Slider.Styles>
+        <i:Interaction.Behaviors>
+          <behaviors:ExecuteCommandOnKeyDownBehavior Command="{Binding MoveSliderRightCommand}" Key="Right"/>
+          <behaviors:ExecuteCommandOnKeyDownBehavior Command="{Binding MoveSliderLeftCommand}"  Key="Left"/>
```

Comment:
Please bind the `IsActive` to the behavior's `IsEnabled` to prevent future issues.
`IsActive` is only `true` when the dialog is the topmost. We have compact dialogs which can be above the normal dialog.

Reply:
@soosr done